packages/core/src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -447,4 +471,18 @@ export default class BraintreeHostedForm {
             errors: this._mapValidationErrors(event.fields),
         });
     };
+
+    private _isValidParam(k: string): k is BraintreeFormErrorDataKeys {
```

Comment:
Nice work, just one comment can we use a more meaningful name than `k`?

Reply:
Good call, let me update this 👍